src/applications/vaos/services/healthcare-service/index.js

```diff
@@ -65,3 +65,45 @@ export async function getSupportedHealthcareServicesAndLocations({
 
   return results.filter(item => item.resourceType === 'Location');
 }
+
+/**
```

Comment:
This code was moved from the transformers file. No changes except to JSDoc comments